packages/cfd/src/Components/__tests__/cfd-dxtrade-demo-account-display.spec.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { TCFDPlatform } from '../props.types';
+import CFDDxtradeDemoAccountDisplay from '../cfd-dxtrade-demo-account-display';
+
+const mocked_props = {
+    has_cfd_account_error: false,
+    standpoint: {
+        financial_company: 'test',
+        gaming_company: 'test',
+        iom: false,
+        malta: true,
```

Comment:
There should not be a mention of these anymore

Reply:
But this was in the file for which I wrote the tests (_packages/cfd/src/Components/cfd-dxtrade-demo-account-display.tsx_): the parent component receive this prop (`standpoint`, `type TStandPoint`) and even use it for checking (line 93,` is_disabled={has_cfd_account_error || standpoint.malta}`). 
I'll remove it from my tests, but the main file (_cfd-dxtrade-demo-account-display.tsx_) should be refactored I think

Reply:
yes it is being refactored separately.